### PR TITLE
Enables different host for the Werkzeug server

### PR DESCRIPTION
**Pull request by ericovis**
I like to configure my projects to run on Docker so I can easily test things locally by adding the missing pieces (database, cache, etc) to my docker-compose.yml file. In short, I needed to make some changes to make your package work inside of a container.

I LOVED your project because it is simple and effective, plus you already have a feature to load ENV vars described in the serverless.yml file inside the dev environment and it saves me some copy and paste. 

I wanted to take advantage of that and also continue to use Docker for my dev environment, however, when I was setting my Docker environment I realized that I couldn't span up a server to run inside a container because your Werkzeug runs the server by default on localhost and there isn't an option to change that. I needed to run the server on '0.0.0.0' in order to make it work and this PR enables this feature.

I hope you can use my work to update yours and thanks for the awesome project 👏 👏 👏

### CODE REVIEW

**Comment by logandk** [comment]
Hi @ericovis, thanks for the PR - excellent work!

I think adding an option to change the host is a good feature, and I'm happy to merge your work on this. I have a few comments that I will add inline on this PR.

File: index.js [hunk 1]
```diff
@@ -180,16 +180,18 @@ class ServerlessWSGI {
...
         path.resolve(__dirname, 'serve.py'),
         this.serverless.config.servicePath,
         this.wsgiApp,
-        port
+        port,
+        host
       ], { stdio: 'inherit' });
       if (status.error) {
-        reject(status.error);
+        reject(status.stdout.toString());
```
Comment (logandk) [inline index.js]: On line 192 we have `{ stdio: 'inherit' }`, so I'm curious about this change. Did you have any problems with the previous implementation? From my testing, `status.stdout` may be null and throws when `toString()` is called, and the output is already emitted to the terminal due to the `inherit` setting anyway.

Comment (ericovis) [reply index.js]: Hey, please don't mind my rudimentary javascript skills.

I was having issues on specifying an alias to the --host parameter, for some reason I couldn't understand the command was failing and returning the help each time I was running it with the previous alias (-h).

I've changed this line when I was troubleshooting the reason and forgot to switch back to the original one. 

BTW I haven't  noticed the `{ stdio: 'inherit' }` so now I feel even stupider for added the status.stdout.toString() 😕 

Bottom line, we should change that to the original value.

Comment (logandk) [reply index.js]: 👍 No problem, thanks for the clarification - I'll update it before merging
